docs/check_spelling.sh

```diff
@@ -0,0 +1,185 @@
+#!/bin/bash
+set -e
+
+# This script does spell checks and can add words to a dictionary.
+# It can be used as a git hook to perform checks, it should be named 
+# "pre-push","pre-commit" based on the use case and kept into the ".git/hooks/"
+# and be executable.It can be bypassed by--no-verify option with git command.
+
+# Language of the doc, install appropriate aspell libs when using 
+# a non-English language
+lang=en
+
+project_dict="./.aspell.en.pws"
+
+# Extension for files to be spell checked
+file_extension=rst
```

Comment:
If it is possible to use several extensions, there are some index files that are called `.rsti` in the blueprint.

Reply:
Now supports multiple extensions